osutil/strace/strace.go

```diff
@@ -50,8 +50,8 @@ var ExcludedSyscalls = getExcludedSyscalls()
 
 func findStrace(u *user.User) (stracePath string, userOpts []string, err error) {
 	if path := filepath.Join(dirs.SnapMountDir, "strace-static", "current", "bin", "strace"); osutil.FileExists(path) {
-		// strace-static cannot resolve usernames, pass uid/gid instead
-		return path, []string{"--uid", u.Uid, "--gid", u.Gid}, nil
+		// Strace v6.9 supports -u UID:GID which avoids the need to resolve usernames with nss.
```

Comment:
Since it required v6.9 just help me understand, do we bundle strace-static or how are we fulfilling this requirement?

Reply:
Ah right, looking more at this it needs the snap to be installed before this works?

Reply:
There are two cases:

- you have the `strace-static` snap, just refresh or it doesn't work
- you don't have the snap - this never worked then

There are about 200-300 users of strace-static, I think it's fine to do it without any extra complexity.